qiita_pet/test/test_admin_processing_job_handlers.py

```diff
@@ -62,13 +62,15 @@ def test_post(self):
         body = response.body.decode('ascii')
         for name in snames:
             self.assertIn(name, body)
+
         # Check failure: invalid qiita id
         post_args = {
             'qid': 2,
             'snames': 'SKB1.640202 SKB2.640194 BLANK.1A BLANK.1B'
         }
         response = self.post('/admin/sample_validation/', post_args)
-        self.assertEqual(response.code, 500)
+        self.assertEqual(response.code, 200)
```

Comment:
Just to confirm, you're changing the response returned from 500 to 200. This is okay, right?

Reply:
Yup, that's basically what this PR is solving a 500 error is not actually reported to the user - this is based on the configuration we have in nginx and to force that the errors are actually nicely catch/parse.